src/lib/Libdb/pgsql/db_attr.c

```diff
@@ -40,7 +40,7 @@
 
 
 /**
- * @file    db_postgres_attr.c
+ * @file    db_attr.c
```

Comment:
We should consider removing the @file tag since we need to change everytime we rename a file. Just like we do not name our functions and doxygen knows automatically, removing the @file tag makes sense to me. Perhaps we can do this for all files u are touching?